python/pyspark/ml/tuning.py

```diff
@@ -868,12 +868,12 @@ class TrainValidationSplitModel(Model, _TrainValidationSplitParams, MLReadable,
     .. versionadded:: 2.0.0
     """
 
-    def __init__(self, bestModel, validationMetrics=[], subModels=None):
+    def __init__(self, bestModel, validationMetrics=None, subModels=None):
```

Comment:
Annotation

https://github.com/apache/spark/blob/46ad325e56abd95c0ffdbe64aad78582da8c725d/python/pyspark/ml/tuning.pyi#L174

should change to `validationMetrics: Optional[List[float]] = ...,`